Cargo.toml

```diff
@@ -136,4 +138,4 @@ vergen = { version = "8", default-features = false }
 void = "1.0.2"
 warp = "0.3.7"
 zeroize = "1.6.0"
-zstd = "0.12.4"
+zstd = "0.13.1"
```

Comment:
This seems to be unrelated?

Reply:
Ok it looks like I have updated cargo.lock while rebasing which lead to a build error that forced this upgrade. All of that is reverted now.